docs/data/gridftp.md

```diff
@@ -83,22 +141,38 @@ This is usually one XML node spread over multiple lines. Note that comments (\#)
 
 #### Selecting a collector host ####
 
-The collector is the central server which logs the GridFTP transfers into a database. There are usually three options:
+The collector is the central server which logs the GridFTP transfers into a database. There are usually two options:
 
 1. **OSG Transfer Collector**: This is the primary collector for transfers in the OSG. Use `CollectorHost="gratia-osg-prod.opensciencegrid.org:80"`.
 1. **OSG-ITB Transfer Collector**: This is the test collector for transfers in the OSG. Use `CollectorHost=" gratia-osg-itb.opensciencegrid.org:80"`.
 
-#### Validation ####
+### Starting the service
+Once the configuration file is edited, as described above, we will proceed to start the service.
+
+for CentOS 7:
+
+```console
+systemctl start gratia-probes-cron
+```
+
+for CentOS 6:
+```console
+service gratia-probes-cron start
+```
+
+### Validation
 
 Run the Gratia probe once by hand to check for functionality:
 
 ``` console
-root@host # /usr/share/gratia/gridftp-transfer/GridftpTransferProbeDriver
+root@host # /usr/share/gratia/gridftp-transfer/gridftp-transfer_meter
 ```
 
 Look for any abnormal termination and report it if it is a non-trivial site issue. Look in the log files in `/var/log/gratia/<date>.log` and make sure there are no error messages printed.
 
-### Optional configuration
+Optional configuration
```

Comment:
Let's keep this as a sub-heading of the `Configuring GridFTP` section

Reply:
Done